Spine/Routing.swift

```diff
@@ -60,12 +59,12 @@ public class Router: RouterProtocol {
 		return baseURL.URLByAppendingPathComponent(type)
 	}
 	
-	public func URLForRelationship<T: ResourceProtocol>(relationship: Relationship, ofResource resource: T) -> NSURL {
+	public func URLForRelationship<T: Resource>(relationship: Relationship, ofResource resource: T) -> NSURL {
 		let resourceURL = resource.URL ?? URLForResourceType(resource.dynamicType.resourceType).URLByAppendingPathComponent("/\(resource.id!)")
```

Comment:
Found an issue. I have 2 default environments (Debug, Release) and another one I created (Staging), according to image below:

<img width="361" alt="screen shot 2015-10-28 at 11 36 07 pm" src="https://cloud.githubusercontent.com/assets/70312/10807952/dfa9c0b6-7dcc-11e5-8173-ea96de462287.png">

When I run a POST/PUT request in my mobile device using the Build Configuration `Staging`, I get this:

<img width="1001" alt="screen shot 2015-10-28 at 11 34 12 pm" src="https://cloud.githubusercontent.com/assets/70312/10807966/0fb861b8-7dcd-11e5-8ad8-dbec7b9a43ed.png">

When I use `Debug`, it works fine.

I do `GET`'s just fine. Seems similar to https://github.com/wvteijlingen/Spine/pull/32. Any ideas?

I did the following and it worked:

```
URLForResourceType((resource as! Resource).dynamicType.resourceType).URLByAppendingPathComponent("/\(resource.id!)")
```